osfoffline/views/start_screen.py

```diff
@@ -1,12 +1,22 @@
+import furl
+import aiohttp
```

Comment:
Thanks. Sadly, while visual appeal is fun in personal code, this is about maintainability- triangular imports don't provide the level of predictability needed for code worked on by many people.

As one example, I recently reviewed a module that used `import module`, `from module import x`, and `import module as alias` in the same block. Had imports been ordered, that clutter could have been avoided.

If it makes you feel better, it's the wave of the future; Jetbrains recently made it the [default behavior](http://blog.jetbrains.com/pycharm/2015/09/announcing-the-pycharm-5-eap-opening/) in one of their products.